avro-builder/tests/tests-allavro/src/test/java/com/linkedin/avroutil1/builder/FixedTypeTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+
+package com.linkedin.avroutil1.builder;
+
+import com.linkedin.avroutil1.compatibility.RandomRecordGenerator;
+import com.linkedin.avroutil1.compatibility.RecordGenerationConfig;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Arrays;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import vs19.FixedType;
+
+
+public class FixedTypeTest {
+
+  private byte[] getByteArrayFromBAIS(ByteArrayInputStream bais) throws IOException {
+    byte[] array = new byte[bais.available()];
+    bais.read(array);
+
+    return array;
+  }
+
+  @Test
+  public void testRoundTripSerialization() throws Exception {
```

Comment:
nit - this tests represents a usage pattern (direct writing of the bytes value of a fixed type) that we dont expect users to do - you either use avro itself to serialize (writer), o rmaybe you use java serialization as a crutch. this would be part of someone's custom serialization logic that we dont expect to see ?